core/src/main/scala/io/scalac/mesmer/core/module/Module.scala

```diff
@@ -1,24 +1,19 @@
 package io.scalac.mesmer.core.module
 
-import com.typesafe.config.{ Config => TypesafeConfig }
+import io.opentelemetry.instrumentation.api.config.{ Config => OTConfig }
```

Comment:
```suggestion
import io.opentelemetry.instrumentation.api.config.{ Config => OpenTelemetryConfig }
```

or 

```suggestion
import io.opentelemetry.instrumentation.api.config.{ Config => OtelConfig }
```

I'd prefer the non-abbreviated one - less "deciphering" on the developer side.